web-client/src/presenter/actions/AdvancedSearch/getPractitionersByNameAction.ts

```diff
@@ -1,25 +1,38 @@
 import { state } from '@web-client/presenter/app.cerebral';
 
-/* gets practitioners matching the name
- *
- * @param {object} providers the providers object
- * @param {object} providers.applicationContext the application context
- * @param {Function} providers.get cerebral get function
- * @returns {object} contains the practitioners returned from the getPractitionersByNameInteractor use case
- */
 export const getPractitionersByNameAction = async ({
   applicationContext,
   get,
+  props,
+  store,
 }: ActionProps) => {
-  const { practitionerName } = get(
-    state.advancedSearchForm.practitionerSearchByName,
+  const {
+    lastKeysOfPages,
+    practitionerName,
+  }: {
+    practitionerName: string;
+    pageNum: number;
+    lastKeysOfPages: Array<string | number>;
+  } = get(state.advancedSearchForm.practitionerSearchByName);
+
+  store.set(
```

Comment:
Nit: I would kind of expect the state setting on line 18 and 30 to be in a different action(s) as we try to keep actions small units of work. I may be misunderstanding though, are there other times we would need to set these state values?

Reply:
It looks like the action is only used in a single sequence right now, however I could see the benefit of splitting this up. Will do.

Reply:
Going to fix this while it's testing.